src/status_im/ui/screens/profile/user/views.cljs

```diff
@@ -220,12 +219,8 @@
                                                                          has-picture)}])
                               :color     (user-avatar.style/customization-color customization-color
                                                                                 (theme/get-theme))
-                              :title     (multiaccounts/displayed-name account)
-                              :photo     (multiaccounts/displayed-photo account)
+                              :title     display-name
+                              :photo     (contact.core/displayed-photo account)
```

Comment:
i feel a little strange here since we're using the namespace `contact.core` to get a profile photo

Reply:
Fixed!